### PR TITLE
Fix inconsistent sorting of department scores

### LINKED ISSUE #4: Inconsistent Score Ranking on Home vs Report Page
Overview:
---

The rankings reverse when you go from nationwide to state pages. It should be the lowest scores at the top for both types of pages. Also, the departments with incomplete scores should show up with the asterisk on the state pages

Details:
---

> Info about the Environment:

* **URL**: https://staging.policescorecard.org & https://staging.policescorecard.org/ca

Acceptance Criteria:
---

> Issue will be considered complete when the following criteria are met:

- [ ] Order should be Lowest to Highest Score for COMPLETE data
- [ ] After COMPLETE data, order should be Lowest to Highest Incomplete Score

Steps to Duplicate:
---

> A step by step guide written for a person who might be looking at this for the first time:

1. Check the Department Scores on the Home Page
2. Check the Department Scores on a Report Page ( specific to a location )
3. You will see they are not rendered the same way

Documentation:
---

> Screenshots, Attachments, Linked GitHub Issues, etc

image

image

Dev & QA Notes:
---

> Any additional notes from developers or QA required to complete this issue.

### PR DESCRIPTION
**Pull request by manifestinteractive**
Overview:
---

Also updated max departments from 500 to 1,000.

Reviewer:
---

> Where should the reviewer start? How to Test? Background Context? etc ( required )

Home page sorting was correct, but report pages had the order backwards.

Checklist:
---

> I have tested each of the following, and they work as expected: ( required )

- [X] Meets [Contributing Guide](https://github.com/campaignzero/police-scorecard/blob/develop/.github/CONTRIBUTING.md) Requirements
- [X] Pulled in the Latest Code from the `develop` branch
- [X] Works on a Desktop / Laptop Device
- [X] Works on a Mobile Device
- [X] `npm test` Does Not Generate Errors

Documentation:
---

> Screenshots, Attachments, Linked GitHub Issues, etc ( optional )

Fixes #4 & #8 

#### What GIF best describes this PR or how it makes you feel?

> Drag & Drop Something Fun Here ( optional )

that_a_boy

### CODE REVIEW

**Review by MasterInTraining23 — CHANGES_REQUESTED** [review]

File: app/View/Components/Partial/Grades.php [hunk 1]
```diff
@@ -16,8 +16,8 @@ class Grades extends Component
      */
     public function __construct($grades = [], $type = 'police-department')
     {
-        // Cap Grades Report to worst 500
-        $this->grades = array_slice($grades, 0, 500);
+        // Cap Grades Report to worst 1000
+        $this->grades = array_slice($grades, 0, 1000);
```
[the remark this answers is not in the document]
Comment (manifestinteractive) [reply app/View/Components/Partial/Grades.php:20]: Class variable adds one time use as a permanent property, which bloats the class.  This is just truncating the data in the constructor and that variable will never be used again.  Happy to change if you still want, but this was intentional.

File: routes/web.php [hunk 2]
```diff
@@ -142,6 +142,11 @@
     $total = getStateTotal($states, $state);
     $location = $scorecard['agency']['slug'];
 
+    // Sort Grades
```
[the remark this answers is not in the document]
Comment (manifestinteractive) [reply routes/web.php:145]: probably makes sense to do this on the API call itself where it is getting this data.  I was just being lazy, good call.

File: app/View/Components/Partial/Grades.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (MasterInTraining23) [reply app/View/Components/Partial/Grades.php:20]: For me, the readability benefits of a static variable outweigh the memory alloc in this scenario. However, I think it's totally fine to leave it as is and simply remove the comment.

**Comment by manifestinteractive** [comment]
@MasterInTraining23 this is ready for you to review again.  Cleaned up everything as requested.

**Review by MasterInTraining23 — APPROVED** [review]

**Pull request merged**